packetfence.cron.d

```diff
@@ -6,6 +6,9 @@ PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
 # Database backup and maintenance script
 30 00 * * * root /usr/local/pf/addons/database-backup-and-maintenance.sh
 
+# Renew any Let's Encrypt certificates the first of the month
+1 0 1 * * root /usr/local/pf/bin/pfcmd renew_lets_encrypt
```

Comment:
Is root necessary or could it run under pf?

Reply:
pfcmd usually always runs as root but the code will "downgrade" to PF rights when performing the update:
https://github.com/inverse-inc/packetfence/pull/4017/files#diff-83d8609c0217f8d78d579e0b28b2295aR78